src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -978,6 +1007,40 @@ def has_args(call_node: ast.Call, required_arg_ids: Tuple[str, ...]) -> bool:
     return False
 
 
+def merge_entry_points(
+    *all_entry_points_with_descriptions_of_source: Tuple[str, Dict[str, Dict[str, str]]]
+) -> Dict[str, List[str]]:
+    """Merge all entry points, throwing ValueError if there are any conflicts."""
+    # keep source_name info for each entry_point until we have merged all
+    # sources, so we can provide better error messages in case of conflicts
```

Comment:
I think you can probably get rid of this comment. It's not clear what `source_name` refers to, and I didn't anticipate you're going to remove the `description_of_source` - presumably it will be used, which this comment is saying fwict.

Reply:
yea, source_name could perhaps better be described as "description_of_source" to reflect the naming of the input arg. Was working on rephrasing the comment, but gave up ;) 
If the code is easy enough to follow, it's not needed any way..